doc/contributing/README.md

```diff
@@ -1,38 +1,27 @@
-# Contributing to Bundler
+# Bundler Contributor Guidelines
 
-Welcome to Bundler! We are so happy that you're here. We know joining a new open source project can be daunting, so here's a quick overview of what you can expect from this documentation.
+Thank you for your interest in making Bundler better! We welcome contributions from everyone. Dozens of contributors like you have submitted feature improvements, fixed bugs, and written new documentation. [Join the Bundler Slack community](http://slack.bundler.io/) to connect with the Bundler core team and other contributors like you.
 
-*Something missing?* Send us a [pull request](../development/PULL_REQUESTS.md)!
+Before submitting a contribution, please take a moment to read through the following guidelines:
 
-**Recommended first steps**
+* [Bundler Code of Conduct](https://github.com/bundler/bundler/blob/master/CODE_OF_CONDUCT.md) (By participating in Bundler, you agree to abide by the terms laid out in the CoC.)
+* [Development Setup Guidelines](https://github.com/bundler/bundler/blob/master/doc/development/SETUP.md)
```

Comment:
Development Setup Guidelines is more of instructions on how to get setup than guidelines. As such, it is the odd one in this list. I presume this list is a set of high level guidelines about conduct of conduct, guidelines on posting issues and PR's, not how to setup your machine to develop.

I'd suggest finding another place for this bullet.